scripts/rollupLicensePlugin.mjs

```diff
@@ -0,0 +1,126 @@
+// @ts-check
```

Comment:
this file needs to be js because it is imported from `packages/vite/rollup.plugin.ts` and it won't be transpiled.